R/utils.R

```diff
@@ -44,3 +44,32 @@ make_lone_dataset <- function(metacore, dataset_name) {
   }
   metacore
 }
+
+#' Check if messages should be displayed
+#' @param verbose Verbosity level
+#' @noRd
+should_message <- function(verbose) {
```

Comment:
would it be better to use a verb for the functions, like check_message, check_warn, etc

should just seems a little strange to me???